app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java

```diff
@@ -79,6 +82,8 @@
 
     private static final long FIRST_PAGE = 1;
 
+    public ComparableItemListImpl<Attendee> itemListImpl;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'itemListImpl' must be private and have accessor methods.](https://app.codacy.com/manual/fossasia/open-event-orga-app/pullRequest?prid=4765425)